SU-HSDO/suhumsci
# STN-734:  Soundcloud Embed needs to expand length of the container it is placed within

# READY FOR REVIEW

## Summary
When a Soundcloud track is added to a text area component using the `Embeddable via URL` option, it should span the entire width of the text area container.

## Need Review By (Date)
5/14/21

## Urgency
medium

## Steps to Test
1. In your CLI, run `npm run test` and confirm our tests pass.
2. Create a flexible page. In the main content section, add a text area component.
3. Edit the text area component and add a Soundcloud track by clicking on the image icon to add `Media`.
4. Click on the `Embeddable` tab. Copy and paste the following Soundcloud URL to the `Add Embeddable via URL` input. ```https://soundcloud.com/mantasssss/dropkillers-batman```
<img width="945" alt="soundcloud" src="https://user-images.githubusercontent.com/12678977/118132894-3217fd80-b3ce-11eb-89a7-ad6e20491ec4.png">

5. Click on the `Add` button. A modal will pop-up requesting you name the track. Enter a name then Save and Click `Insert Selected`.
6. Confirm the Soundcloud track spans 100% of the text area.
7. Test that the Soundcloud spans 100% of the width in all the following scenarios:
- [x] Admin logged in / logged out states
- [x] Traditional / Colorful themes
8. Confirm the Soundcloud embed is keyboard accessible.
9. Test that the changes in this PR do not change how other embeds display. Add an embed for Twitter, Issuu, Flickr, YouTube, Instagram, Google Forms, CodePen, etc. A list of embeds is available on https://sparkbox-sandbox-prod.stanford.edu/qa/qa-embeds.
